Assets/Scripts/InventoryObject.cs

```diff
@@ -7,10 +7,18 @@ public class InventoryObject : InteractiveObject
     [SerializeField]
     private string objectName = nameof(InventoryObject);
 
+    [SerializeField]
+    [TextArea(3,8)]
+    private string description;
+
+    [SerializeField]
+    private Sprite icon;
     // TODO: Add long description field
```

Comment:
You can get rid of these TODO comments since they're done.

Reply:
Fixed